packages/devextreme/js/__internal/ui/chat/chat_avatar.ts

```diff
@@ -0,0 +1,57 @@
+import $ from '@js/core/renderer';
+import type { WidgetOptions } from '@js/ui/widget/ui.widget';
+
+import Widget from '../widget';
+
+const CHAT_MESSAGE_AVATAR_CLASS = 'dx-chat-message-avatar';
+const CHAT_MESSAGE_AVATAR_LETTERS_CLASS = 'dx-chat-message-avatar-letters';
+
+export interface AvatarOptions extends WidgetOptions<Avatar> {
+  name?: string;
+}
+
+class Avatar extends Widget<AvatarOptions> {
+  _getDefaultOptions(): AvatarOptions {
+    return {
+      ...super._getDefaultOptions(),
+      name: '',
+    };
+  }
+
+  _getAvatarInitials(name: string): string {
```

Comment:
Your function _getAvatarInitials extracts the first letter of a given name, why we use plural form?

Reply:
We don't have a requirement for letters count right now. The function is so named because initials is a generalized word for any number of first letters

Reply:
Well, what about $letters. 
```suggestion
   if (name) {
      const text = this._getAvatarInitials(name);

      $letters.text(text);
    }
```
According to name of this variable I am expecting more that one char.

Reply:
[d4b674e8e7ab763f1263b0db5194a4c54e2564b9]